starters/next-drupal-starter/pages/index.jsx

```diff
@@ -94,6 +93,8 @@ export async function getServerSideProps(context) {
       throw new Error(
         "No articles returned. Make sure the objectName and params are valid!"
       );
+    } else {
```

Comment:
Not sure about sticking this in an `else`. Also, are we passing the sorted articles into the props? Does simply calling this function on an array mutate the original array?  Perhaps we should create a new array and return that one if that's the case so we can avoid side-effects.